src/dfi/tokens.cpp

```diff
@@ -171,13 +171,23 @@ Res CTokensView::UpdateToken(UpdateTokenContext &ctx) {
         }
     }
 
+    // Remove deprecated prefix before symbol check, will be added if required after.
+    if (newToken.symbol.rfind(CToken::DeprecationPrefix(), 0) == 0) {
```

Comment:
Using `IsDeprecated` here as well might make this more robust. If a token is other named as `eol/`, this result in confusing behavior removing part of the token name itself. 

At the moment however, `/` is special, so it's OK. Perhaps let's just add this as a comment or link to this comment int the PR as note otherwise.